tests/integration/models/reddit/test_modmail.py

```diff
@@ -34,6 +34,28 @@ def test_mute(self, _):
             conversation = self.reddit.subreddit('all').modmail('ik72')
             assert conversation.user.mute_status['isMuted']
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_read(self, _):
+        self.reddit.read_only = False
+        conversation = self.reddit.subreddit('all').modmail('ik72')
+        with self.recorder.use_cassette('TestModmailConversation.test_read'):
+            conversation.read()
+            conversation = self.reddit.subreddit('all').modmail('ik72')
+            assert conversation.last_unread is None
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_read__other_conversations(self, _):
+        self.reddit.read_only = False
+        conversation = self.reddit.subreddit('all').modmail('p8rp')
+        other_conversation = self.reddit.subreddit('all').modmail('p8rr')
+        with self.recorder.use_cassette(
+                'TestModmailConversation.test_read__other_conversations'):
+            conversation.read(other_conversations=[other_conversation])
+            conversation = self.reddit.subreddit('all').modmail('p8rp')
+            other_conversation = self.reddit.subreddit('all').modmail('p8rr')
```

Comment:
Are you sure they don't fetch? I see requests for them in the cassette.

In general, the tests need not really assert that the action actually occurred. I usually manually verify when I write my own cassettes. I personally prefer to only test the action directly in order to keep the cassette simpler. An added benefit of only having a single request to the OAuth endpoint is that there is no need to patch `time.sleep`.

If you want to keep it, however, I'm okay with that too.